fact/analysis/statistics.py

```diff
@@ -39,3 +79,100 @@ def li_ma_significance(n_on, n_off, alpha=0.2):
         return significance[0]
 
     return significance
+
+
+def calc_weight_simple_to_curved(energy, gamma, a, b):
+    '''
+    Reweigt simulated events from a simulated power law with
+    spectral index `gamma` to a curved power law with parameters `a` and `b`
+
+    Parameters
+    ----------
+    energy: float or array-like
+        Energy of the events
+    gamma: float
+        Spectral index of the simulated power law
+    a: float
+        Parameter `a` of the target curved power law
+    b: float
+        Parameter `b` of the target curved power law
+    '''
+    return energy ** (gamma + a + b * np.log10(energy))
+
+
+def calc_weight_change_index(energy, simulated_gamma, target_gamma):
+    '''
+    Reweigt simulated events from one power law index to another
+
+    Parameters
+    ----------
+    energy: float or array-like
+        Energy of the events
+    simulated_gamma: float
+        Spectral index of the simulated power law
+    target_gamma: float
+        Spectral index of the target power law
+    '''
+    return energy ** (target_gamma - simulated_gamma)
+
+
+def calc_gamma_obstime(n_events, gamma, phi_0, max_impact, e_min, e_max):
+    '''
+    Calculate the equivalent observation time for a gamma montecarlo set
+
+    Parameters
+    ----------
+    n_events: int
+        Number of simulated events
+    gamma: float
+        Spectral index of the simulated power law
+    phi_0: float
+        Flux normalization of the simulated power law
+    max_impact: float
+        Maximal simulated impact
+    e_min: float
+        Mimimal simulated energy
+    e_max: float
+        Maximal simulated energy
+    '''
+    if gamma >= -1:
+        raise ValueError('gamma must be < -1')
+
+    numerator = n_events * (1 - gamma)
+
+    t1 = phi_0 * max_impact**2 * np.pi
+    t2 = e_max**(1 - gamma) - e_min**(1 - gamma)
+
+    denominator = t1 * t2
+
+    return numerator / denominator
+
+
+def calc_proton_obstime(n_events, gamma, max_impact, viewcone, e_min, e_max):
+    '''
+    Calculate the equivalent observation time for a proton montecarlo set
+
+    Parameters
+    ----------
+    n_events: int
+        Number of simulated events
+    gamma: float
+        Spectral index of the simulated power law
+    max_impact: float
+        Maximal simulated impact
+    viewcone: float
+        Viewcone in degrees
+    e_min: float
+        Mimimal simulated energy in GeV
+    e_max: float
+        Maximal simulated energy in GeV
+    '''
+    N0 = 1.8e4
```

Comment:
flux normalisation from PDG ? if so, please cite and give units maybe? Maybe also make it a default parameter